wotlk-deathknight.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't think you're supposed to commit this file  :)

Also looks like a `wowsimwotlk` binary file is being committed, that should be removed too.